ospfd/ospfd.c

```diff
@@ -448,6 +448,11 @@ struct ospf *ospf_lookup_by_vrf_id(vrf_id_t vrf_id)
 	vrf = vrf_lookup_by_id(vrf_id);
 	if (!vrf)
 		return NULL;
+	return ospf_lookup_by_vrf(vrf);
```

Comment:
Why do we even need a ospf_lookup_by_vrf_id function anymore?

Reply:
in separate commit, this should be done. actually, a work of optimisation should be done for each daemon.